### ISSUE TITLE
Problem with download videos from mirror.co.uk/video

### ISSUE THREAD

**Issue by hichamox** [rg3/youtube-dl#6969]
hey guys first of all thank you for this amazing app it really useful, but unfortunately i cant download videos from http://www.mirror.co.uk/video/ i wish if you can fix this problem also make new option that can give me the download link directly for example site.com/video.mp4.

thank you

**Comment by yan12125** [comment]
Which video are you trying to download?

**Comment by dstftw** [comment]
Post [an example URL](https://github.com/rg3/youtube-dl/blob/master/README.md#is-the-description-of-the-issue-itself-sufficient).

**Comment by hichamox** [comment]
this one http://www.mirror.co.uk/news/weird-news/forget-pizza-rat-watch-milkshake-6498785

**Comment by TRox1972** [comment]
Seems to be using brightcove.

**Comment by yan12125** [comment]
Yeah, and it's using a custom script to load the Brightcove video.
